.github/code_owners.yml

```diff
@@ -84,6 +84,8 @@ examples/**:
 # docs
 docs/**:
 - tylerbutler
+"**/*.md":
```

Comment:
i think the quotes will prevent this from matching as expected

Reply:
The documentation for the action has an example like this. But I should do some testing on my fork... I'll do that and merge if everything works.

Reply:
Did it work :O The docs said it follows the same formatting as the gitignore, and I had weird behavior with that using quotes

Reply:
It did work! https://github.com/tylerbutler/FluidFramework/runs/1178752176?check_suite_focus=true#step:2:33

Ultimate it failed on that run because it's my fork, but it detected the changed path correctly and tried to assign it properly.